signbank/dictionary/csv_import.py

```diff
@@ -261,6 +275,57 @@ def import_nzsl_share_gloss_csv(request):
                   })
 
 
+def update_retrieval_videos(videos, gloss_data, gloss):
```

Comment:
A note on the (somewhat) doubled-up arguments. This is how this existing code worked and I didn't want to investigate making major refactoring changes to it, though I have made a couple of small mods.  I just needed to encapsulate the population of the `videos` list so that it could be called more than once.

Reply:
I have changed this so there is now only one list passed in.